### PR TITLE
ask confirmation on init command

### PR DESCRIPTION
**Pull request by skv-headless**
if directory already exist.

### CODE REVIEW

File: react-native-cli/index.js [hunk 1]
```diff
@@ -65,6 +66,30 @@ function init(name) {
...
+    var property = {
+      name: 'yesno',
+      message: 'Directory ' + name + ' already exist. Continue?',
+      validator: /y[es]*|n[o]?/,
+      warning: 'Must respond yes or no',
+      default: 'no'
+    };
+
+    prompt.get(property, function (err, result) {
+      if (result.yesno === 'yes') {
```
Comment (frantic) [inline react-native-cli/index.js]: your regex allows `y` and `n`, but the rest of code requires `yes`. Maybe check `yesno[0] === 'y'`?

File: react-native-cli/index.js [hunk 2]
```diff
@@ -65,6 +66,30 @@ function init(name) {
...
+        createProject(name);
+      } else {
+        process.exit(1);
```
Comment (frantic) [inline react-native-cli/index.js]: Would be nice to show a message that init was canceled

**Comment by frantic** [comment]
Thanks! Please fix these 2 nits and we'll get it in.

**Comment by skv-headless** [comment]
@frantic done.

**Pull request merged**